community/kernel/src/main/java/org/neo4j/storageengine/api/schema/IndexSampler.java

```diff
@@ -20,24 +20,19 @@
 package org.neo4j.storageengine.api.schema;
 
 import org.neo4j.kernel.api.exceptions.index.IndexNotFoundKernelException;
-import org.neo4j.register.Register;
 
 /**
  * Component able to sample schema index.
  */
 public interface IndexSampler
 {
-    IndexSampler EMPTY = result -> {
-        result.write( 0, 0 );
-        return 0;
-    };
+    IndexSampler EMPTY = () -> new IndexSample( 0, 0, 0 );
```

Comment:
Can we please introduce default constructor in `IndexSample` with 0 as default value for all sizes/values and replace all usages of `new IndexSample(0, 0, 0)` with `new IndexSample()`?